webpack.dev.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+const {merge} = require('webpack-merge');
+const base = require('./webpack.config');
+
+module.exports = merge(base, {
+    
+    mode: 'development',
+
+    output: {
+        path: path.resolve(__dirname, './distDev'),
+        filename: 'dev.bundle.js',
+    },
```

Comment:
```suggestion
    output: {
        path: path.resolve(__dirname, './dist),
        filename: 'dev.bundle.js',
    },
```

Сделай пожалуйста для дев и прод одну папку билда и помести `output` в базовый конфиг